static/templates/settings/display-settings.handlebars

```diff
@@ -101,10 +90,14 @@
                     <input type="radio" class="emojiset_choice" name="emojiset_group" value="{{@key}}" />
                     <span>{{this}}</span>
                     <span class="right">
+                        {{#if @last}}
+                        <div class="emoji_alt_code">&nbsp:relaxed:</div>
```

Comment:
Can you increase the font size a bit more for this. This looks too small right now.